_docs/tasks/installing-istio.md

```diff
@@ -0,0 +1,140 @@
+
+    ```bash
+    gcloud config set container/use_client_certificate True
+    gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <project-name>
+    ```
+
+* Ensure the `curl` command is present.
+
+## Installing on an existing cluster
+
+For the Alpha release, Istio must be installed in the same Kubernetes namespace as the applications. Instructions below will deploy Istio in the
+default namespace. They can be modified for deployment in a different namespace.
+
+1. Download and extract the [istio installation files](https://raw.githubusercontent.com/istio/istio/master/releases/istio-alpha.tar.gz), or
+clone Istio's [GitHub](https://github.com/istio/istio) repository:
+
+    ```bash
+    git clone https://github.com/istio/istio
+    ```
+
+2. Change directory to istio:
+
+    ```bash
+    cd istio
+    ```
+
+3. Install Istio's core components (Istio-Manager, Mixer, and Ingress-Controller):
+
+    ```bash
+    kubectl apply -f ./kubernetes/istio.yaml
+    ```
+
+4. Source the Istio configuration file:
+
+    ```bash
+    source istio.VERSION
+    ```
+
+5. Download one of the [`istioctl`](/docs/reference/istioctl.html) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
+`istioctl-linux`, targeted at Mac, Windows or Linux users respectively. For example, run the following commands on a Mac system:
+
+    ```bash
+    curl ${ISTIOCTL_URL}/istioctl-osx > /usr/local/bin/istioctl
+    chmod +x /usr/local/bin/istioctl
+    ```
+
+    `istioctl` is needed to inject Envoy as a sidecar proxy. It also provides a convenient CLI for creating routing rules and policies.
+    Note: If you already have a previously installed version of `istioctl`, make sure that
+    it is compatible with the manager image used in `istio.yaml`.
+    If in doubt, download again or add the `--tag` option when running `istioctl kube-inject`.
+    Invoke `istioctl kube-inject --help` for more details.
+
+4. Deploy your application with Envoy:
+
+    ```bash
+    kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
+    ```
+
+    The [kube-inject](/docs/reference/istioctl.html##kube-inject) tool will automatically inject an Envoy container in the pod running the application.
+    Alternatively, deploy one of the sample applications, for instance [bookinfo](/docs/samples/bookinfo.html).
+
+6. Optionally: to view metrics collected by Mixer, install [Prometheus](https://prometheus.io), [Grafana](http://staging.grafana.org) or
+ServiceGraph addons:
+
+    ```bash
+    kubectl apply -f ./kubernetes/addons/grafana.yaml
+    kubectl apply -f ./kubernetes/addons/prometheus.yaml
+    kubectl apply -f ./kubernetes/addons/servicegraph.yaml
+    ```
+
+    The Grafana image provided as part of this sample contains a built-in Istio dashboard that you can access from:
+
+    ```bash
+    http://<grafana-svc-external-IP>:3000/dashboard/db/istio-dashboard
```

Comment:
These instructions expect the user to know the grafana-svc-external-IP which has not been introduced yet. They expect the user to understand how to change the Service from type LoadBalancer to type NodePort and under what circumstances that is needed. They expect the user to know how to run _kubectl proxy_ and how to edit Grafana's Istio-dashboard.